### PR TITLE
oc_image to lib_openshift.

### PR DESCRIPTION
**Pull request by kwoodson**

### CODE REVIEW

**Comment by kwoodson** [comment]
@ivanhorvath, Updated and opened PR.  Let me know if you have any questions/concerns.

File: roles/lib_openshift/src/doc/image [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oc_label
+short_description: Create, modify, and idempotently manage openshift labels.
+description:
+  - Modify openshift labels programmatically.
+options:
+  state:
```
Comment (ashcrow) [inline roles/lib_openshift/src/doc/image]: namespace is missing

Comment (kwoodson) [inline roles/lib_openshift/src/doc/image]: Thanks, fixing

**Comment by kwoodson** [comment]
aos-ci-test

**Comment by kwoodson** [comment]
aos-ci-test

**Comment by kwoodson** [comment]
[merge]

**Pull request merged**